examples/compose/lite/authelia/configuration.yml

```
@@ -5,8 +5,7 @@
 
 host: 0.0.0.0
 port: 9091
-log:
-  level: debug
+log_level: debug
```

```suggestion
log:
  level: debug
```

Please remove this change. The newest version of Authelia will be deprecating the old configuration method. We should instead make sure the documentation for the lite bundle has people checkout the latest tagged version of the repository. See an example here: https://www.authelia.com/docs/getting-started.html#steps